x-pack/plugins/transform/public/app/sections/create_transform/components/step_details/step_details_form.tsx

```diff
@@ -500,6 +539,86 @@ export const StepDetailsForm: FC<Props> = React.memo(
           )}
+
+          <EuiSpacer size="l" />
+
+          <EuiAccordion
+            data-test-subj="transformWizardAccordionAdvancedSettings"
+            id="transformWizardAccordionAdvancedSettings"
+            buttonContent={i18n.translate(
+              'xpack.transform.stepDetailsForm.advancedSettingsAccordionButtonContent',
+              {
+                defaultMessage: 'Advanced settings',
+              }
+            )}
+            paddingSize="s"
+          >
+            <EuiFormRow
+              label={i18n.translate('xpack.transform.stepDetailsForm.frequencyLabel', {
+                defaultMessage: 'Frequency',
+              })}
+              isInvalid={!isTransformFrequencyValid}
+              error={
+                !isTransformFrequencyValid && [
+                  i18n.translate('xpack.transform.stepDetailsForm.frequencyError', {
+                    defaultMessage: 'Invalid frequency format',
+                  }),
+                ]
+              }
+              helpText={i18n.translate('xpack.transform.stepDetailsForm.frequencyHelpText', {
+                defaultMessage:
+                  'The interval between checks for changes in the source indices when the transform is running continuously. Also determines the retry interval in the event of transient failures while the transform is searching or indexing. The minimum value is 1s and the maximum is 1h.',
+              })}
+            >
+              <EuiFieldText
+                placeholder="frequency"
+                value={transformFrequency}
+                onChange={(e) => setTransformFrequency(e.target.value)}
+                aria-label={i18n.translate('xpack.transform.stepDetailsForm.frequencyAriaLabel', {
+                  defaultMessage: 'Choose a frequency.',
+                })}
+                isInvalid={!isTransformFrequencyValid}
+                data-test-subj="transformFrequencyInput"
+              />
+            </EuiFormRow>
+
+            <EuiFormRow
+              label={i18n.translate('xpack.transform.stepDetailsForm.maxPageSearchSizeLabel', {
+                defaultMessage: 'Maxiumum page search size',
+              })}
+              isInvalid={!isTransformSettingsMaxPageSearchSizeValid}
+              error={
+                !isTransformSettingsMaxPageSearchSizeValid && [
+                  i18n.translate('xpack.transform.stepDetailsForm.maxPageSearchSizeError', {
+                    defaultMessage: 'Invalid max_page_search_size value',
+                  }),
+                ]
+              }
+              helpText={i18n.translate(
+                'xpack.transform.stepDetailsForm.maxPageSearchSizeHelpText',
+                {
+                  defaultMessage:
+                    'Defines the initial page size to use for the composite aggregation for each checkpoint.',
+                }
+              )}
+            >
+              <EuiFieldText
+                placeholder="max_page_search_size"
```

Comment:
Can this be made the same placeholder text as in the edit form which is much more user-friendly.

![image](https://user-images.githubusercontent.com/7405507/93221303-84345080-f765-11ea-87e0-8ddc423bb358.png)

If you delete the value in the wizard you get:

![image](https://user-images.githubusercontent.com/7405507/93221388-a4640f80-f765-11ea-861e-5a1508724df8.png)

Reply:
Updated the placeholder in 01d852b8b37. I will look into fixing the `NaN` issue in a follow up when consolidating the form code. Added a note to the meta issue here: https://github.com/elastic/kibana/issues/77604